docs/content/config.md

```diff
@@ -4,61 +4,61 @@ description: "use workspacer like a pro!"
 type: faq
 ---
 
-Workspacer is configured with the C# programming language. The expressiveness of C# allows you to be hyper-specifc in terms of describing your desired behavior.
-This configuration documentation assumes that you have already created the example config in the correct folder, if you haven't, check the [quickstart guide](/quickstart).
+Workspacer is configured with the C# programming language. The expressiveness of C# allows you to be hyper-specific in terms of describing your desired behavior.
+This configuration documentation assumes that you have already created the example config in the correct folder, if you haven't, check the [quick-start guide](/quickstart).
 
 ## I don't know C#, or how to program, should I use workspacer?
 
 Workspacer is intended to be on the extreme end of the power user spectrum. You can use workspacer without knowing how to program, but extreme configuration will be challenging if you need to deviate outside of the examples and configurations of other users.
```

Comment:
Maybe it will sound better "You can use workspacer without knowing how to code" instead of how to program?

Using extreme here makes it sound a bit too exaggerated, suggestions: further / furthest / major / advanced.

Is it correct to say deviate oustide of something? Deviating from sounds correct in my head but I've never heard about deviating outside of something, at least in this context it doesn't sound good.

Reply:
1. "code" is a bit clearer
2. I agree - I think "advanced" is a good choice
3. Both "deviate from" and "deviate outside" are widely used - I don't think there's any issues there, apart from personal style

Reply:
Sounds good, [done](https://github.com/workspacer/workspacer/pull/229/commits/e3b9edd12c8a4349bcc65fd517e429b965652b65).

Reply:
I didn't change "extreme end", because extreme there sounds accurate, I changed "...but extreme configuration..."

Reply:
Yup, I agree 👍